Documentation/install/system_requirements.rst

```diff
@@ -79,7 +79,7 @@ Linux Kernel
 
 Cilium leverages and builds on the kernel BPF functionality as well as various
 subsystems which integrate with BPF. Therefore, host systems are required to
-run Linux kernel version 4.8.0 or later to run a Cilium agent. More recent
+run Linux kernel version 4.9.17 or later to run a Cilium agent. More recent
```

Comment:
Unrelated to this PR, but yes this is the minimum version we support (Though we've hit other problems in early 4.9.x versions so these days we only test a 4.9.2xx version with the latest fixes).